sci-libs/pcl/pcl-1.12.1-r2.ebuild

```diff
@@ -24,7 +24,7 @@ DESCRIPTION="2D/3D image and point cloud processing"
 LICENSE="BSD"
 SLOT="0/1.12"
 IUSE="cuda doc opengl openni openni2 pcap png +qhull qt5 usb vtk cpu_flags_x86_sse test tutorials"
-RESTRICT="!test? ( test )"
+RESTRICT="test"
```

Comment:
can you add a comment to explain why tests are masked?

Reply:
> * restrict test, because it needs the gtest sources to be provided, the package alone seems not be enough.

It's already there

Reply:
The concrete output is:

```
CMake Error at /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find GTestSource (missing: GTEST_SRC_DIR)
Call Stack (most recent call first):
  /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:594 (_FPHSA_FAILURE_MESSAGE)
  cmake/Modules/FindGTestSource.cmake:35 (find_package_handle_standard_args)
  test/CMakeLists.txt:14 (find_package)


-- Configuring incomplete, errors occurred!
```